docs/src/main/asciidoc/logging.adoc

```diff
@@ -198,6 +198,39 @@ Applications and components may use any of the following APIs for logging, and t
 
 If you want to send your logs to a centralized tool like Graylog, Logstash or Fluentd, you can follow the link:centralized-log-management[Centralized log management guide].
 
+== Pitfalls
+
+=== None of the configuration properties seems to have any effect
+
+You will most likely see two separate log lines in your output:
+----
+Mar 14, 2020 12:47:37 PM com.mongodb.diagnostics.logging.SLF4JLogger info
+INFO: Opened connection [connectionId{localValue:4, serverValue:236}] to 127.0.0.1:27017
+----
+
+This is usually because you have some agent or something which has initialized logging too early. If that's the case, the workaround is to use the following settings.
+
+For unit tests in your IDE:
+[source]
+----
+-Djava.util.logging.manager=org.jboss.logmanager.LogManager
+----
+
+For your Maven configuration:
```

Comment:
Duplicate of the section just below - https://github.com/quarkusio/quarkus/blob/544773f4c97cbebd0e055d069970fc3e519a9a4a/docs/src/main/asciidoc/logging.adoc#how-to-configure-logging-for-quarkustest ?

Reply:
I integrated it now to make it more clear.